Doc/library/io.rst

```diff
@@ -171,22 +171,23 @@ separation between reading and writing to streams; implementations are allowed
 to raise :exc:`UnsupportedOperation` if they do not support a given operation.
 
 The :class:`RawIOBase` ABC extends :class:`IOBase`.  It deals with the reading
-and writing of bytes to a stream.  :class:`FileIO` subclasses :class:`RawIOBase`
-to provide an interface to files in the machine's file system.
-
-The :class:`BufferedIOBase` ABC deals with buffering on a raw byte stream
-(:class:`RawIOBase`).  Its subclasses, :class:`BufferedWriter`,
-:class:`BufferedReader`, and :class:`BufferedRWPair` buffer streams that are
-readable, writable, and both readable and writable.  :class:`BufferedRandom`
-provides a buffered interface to random access streams.  Another
-:class:`BufferedIOBase` subclass, :class:`BytesIO`, is a stream of in-memory
-bytes.
-
-The :class:`TextIOBase` ABC, another subclass of :class:`IOBase`, deals with
-streams whose bytes represent text, and handles encoding and decoding to and
-from strings. :class:`TextIOWrapper`, which extends it, is a buffered text
-interface to a buffered raw stream (:class:`BufferedIOBase`). Finally,
-:class:`StringIO` is an in-memory stream for text.
+and writing of bytes to a stream.  :class:`FileIO` subclasses
```

Comment:
Are you just add and `\n` to this paragraph?

Reply:
I edited my post above to make it clearer what the modifications are as the reformatting of line breaks makes it hard to see.